### DISCUSSION TITLE
Ligature shakes unexpectedly

### DISCUSSION THREAD

**Discussion by zxkyjimmy** [ghostty-org/ghostty#4175] [Q&A]
When I continuously press any printable character on the keyboard, the Python `>>>` prompt keeps shaking.

https://github.com/user-attachments/assets/b73595b0-34ae-405f-8e0d-4959136a517e

**Comment by HofmT** [comment]
I experience the exact same issue. I'm on MacOS Sequoia 15.0 and use the same Python version has the submitter.
The issue is not seen in Python 2.7 only Python 3.


https://github.com/user-attachments/assets/1b55fd76-e0ac-431a-98ce-1d8527b58129
